test/extended/machine_config/machine_config_node.go

```diff
@@ -29,7 +29,7 @@ const (
 	arbiter = "arbiter"
 )
 
-var _ = g.Describe("[sig-mco][OCPFeatureGate:MachineConfigNodes]", func() {
+var _ = g.Describe("[Suite:openshift/machine-config-operator/disruptive][sig-mco][OCPFeatureGate:MachineConfigNodes]", func() {
```

Comment:
As far as we can tell, this would cause the test to only run in the new MCO suite, and not for serial runs anymore.

We're depending on this for signal, so while we're transitioning between the tests, would it be possible to run the test on both Serial and the new MCO suite? Would we have to define this test twice with different labels?